csaf-validator-lib/tests/shared/minimalVexDoc.js

```diff
@@ -0,0 +1,17 @@
+import minimalDoc from './shared/minimalDoc.js'
+
+export default {
+  ...minimalDoc,
+  document: {
+    ...minimalDoc.document,
+    category: 'csaf_vex',
+  },
+  product_tree: {
+    full_product_names: [
+      {
+        product_id: 'CSAFPID-0001',
+        name: 'Some sample product',
+      },
+    ],
+  },
```

Comment:
Actually, that is not a minimal VEX document. It is missing the `/vulnerabilities` section...

Reply:
Could you open an issue in the `csaf-validator-lib` repo for that?

Reply:
Sure. Actually, I wanted to do a review - but I clicked on the wrong button... :facepalm:

Reply:
See https://github.com/secvisogram/csaf-validator-lib/issues/4.